tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -188,6 +188,13 @@ Verify Favorite Resource Cards
     Favorite Items Should Be Listed First When Sorted By    ${favorite_ids}    duration
     [Teardown]    Remove Items From favorites    @{favorite_ids}
 
+Verify Dashboard Pod Is Not Getting Restarted	
+    [Documentation]    Verify Dashboard Pod container restart
+    [Tags]    Sanity
+    ...       ODS-374
+    ${pod_names}    Get POD Names    redhat-ods-applications    ${LABEL_SELECTOR}
```

Comment:
`${LABEL_SELECTOR}` in my opinion is not necessary. If you want to keep it, it could be renamed to` ${DASHBOARD_LABEL_SELECTOR}`